paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodEmbeddedLayoutUIScreenshotTest.kt

```diff
@@ -0,0 +1,181 @@
+                paymentMethods = paymentMethods,
+                displayedSavedPaymentMethod = savedPaymentMethod,
+                savedPaymentMethodAction =
+                PaymentMethodVerticalLayoutInteractor.SavedPaymentMethodAction.MANAGE_ALL,
+                selection = PaymentSelection.Saved(savedPaymentMethod.paymentMethod),
+                isEnabled = true,
+                onEditPaymentMethod = {},
+                onViewMorePaymentMethods = {},
+                onSelectSavedPaymentMethod = {},
+                onManageOneSavedPaymentMethod = {},
+                imageLoader = mock(),
+                rowStyle = PaymentSheet.Appearance.Embedded.RowStyle.FlatWithCheckmark.defaultLight
+            )
+        }
+    }
+
+    @Test
+    fun testSeparatorColorAndInsets() {
+        paparazziRule.snapshot {
+            PaymentMethodEmbeddedLayoutUI(
+                paymentMethods = paymentMethods,
+                displayedSavedPaymentMethod = savedPaymentMethod,
+                savedPaymentMethodAction =
+                PaymentMethodVerticalLayoutInteractor.SavedPaymentMethodAction.MANAGE_ALL,
+                selection = PaymentSelection.Saved(savedPaymentMethod.paymentMethod),
+                isEnabled = true,
+                onEditPaymentMethod = {},
+                onViewMorePaymentMethods = {},
+                onSelectSavedPaymentMethod = {},
+                onManageOneSavedPaymentMethod = {},
+                imageLoader = mock(),
+                rowStyle = PaymentSheet.Appearance.Embedded.RowStyle.FlatWithCheckmark(
+                    separatorThicknessDp = 5f,
+                    separatorColor = Color.CYAN,
+                    separatorInsetsDp = 40f,
+                    topSeparatorEnabled = true,
+                    bottomSeparatorEnabled = true,
+                    checkmarkColor = StripeThemeDefaults.colorsLight.materialColors.primary.toArgb(),
+                    checkmarkInsetDp = StripeThemeDefaults.embeddedCommon.checkmarkInsetDp,
+                    additionalInsetsDp = StripeThemeDefaults.embeddedCommon.additionalInsetsDp
+                )
+            )
+        }
+    }
+
+    @Test
+    fun testBottomSeparatorOnly() {
+        paparazziRule.snapshot {
+            PaymentMethodEmbeddedLayoutUI(
+                paymentMethods = paymentMethods,
+                displayedSavedPaymentMethod = savedPaymentMethod,
+                savedPaymentMethodAction =
+                PaymentMethodVerticalLayoutInteractor.SavedPaymentMethodAction.MANAGE_ALL,
+                selection = PaymentSelection.Saved(savedPaymentMethod.paymentMethod),
+                isEnabled = true,
+                onEditPaymentMethod = {},
+                onViewMorePaymentMethods = {},
+                onSelectSavedPaymentMethod = {},
+                onManageOneSavedPaymentMethod = {},
+                imageLoader = mock(),
+                rowStyle = PaymentSheet.Appearance.Embedded.RowStyle.FlatWithCheckmark(
```

Comment:
Can we make a helper function for these? Maybe just passing in the row style? Might need some other small customizations too.

Reply:
Add a helper function to pass in RowStyle type and optionally adjust defaults